MDANSE/Src/MDANSE/Mathematics/Arithmetic.py

```diff
@@ -19,7 +19,9 @@
 import numpy as np
 
 
-def get_weights(props: Dict[str, float], contents: Dict[str, int], dim: int):
+def get_weights(
+    props: Dict[str, float], contents: Dict[str, int], dim: int, c_exp: float = 1.0
```

Comment:
Not sure I like the name `c_exp` reminds me of the function `cexp(x)` which is `e**(i*x)`. It is, however, documented clearly, so I'm happy to keep it if there's not a better name.

Reply:
Good point, I've changed it to conc_exp.